securitycenter/api/src/Snippets.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
This is why it's asking for a Main method. Check the KMS samples, the project file should be the same except for the actual dependencies.